es-app/src/SystemScreenSaver.cpp

```diff
@@ -301,8 +301,8 @@ void SystemScreenSaver::pickGameListNode(unsigned long index, const char *nodeNa
 		pugi::xml_document doc;
 		pugi::xml_node root;
 
-		// We only want images from game systems
-		if (!(*it)->isGameSystem())
+		// We only want nodes from game systems that are not collections
+		if (!(*it)->isGameSystem() || (*it)->isCollection())
```

Comment:
Sorry, I'm just tired :)

Reading only a snippet of the code, with the comment "we only want nodes that are not collections" but the condition checking whether it _is_ a collection got me to ask about that. I thought the condition was going to check if they were not collections.

Nevermind.

You are indeed using isGameSystem correctly. RetroPie will not be a GameSystem. 

A collection is really just a group of existing games - a subset, organized in a new top level entry (system) - but keeping the games in their original places.

You can easily test it out by turning on the "All Games" and "Last Played" at least on your menu, if you don't want to play with "Favorites".

Thanks - any questions just ask.